source/access/managing_disk_usage.rst

```diff
@@ -5,98 +5,47 @@ How much disk space am I using?
 
-Total disk space used on filesystems with quota
------------------------------------------------
+Total disk space used on file systems with quota
+------------------------------------------------
 
-On filesystems with quota enabled, you can check the amount of disk space that
+On file systems with quota enabled, you can check the amount of disk space that
 is available for you, and the amount of disk space that is in use by
-you.  Unfortunately, there is not a single command that will give
-you that information for all file systems in the VSC.
-
-On most systems, ``quota`` will show you for each file system
-
-``space``
-   The disk space taken up by your files.
-``quota``
-   If the disk space you are using exceeds this value, you may
-   get warnings (disabled on most systems).  The reported disk
-   usage will be marked with an ``*`` if it exceeds the value
-   of ``quota``.
-``limit``
-   If the limit is reached, nothing can be written to disk, so
-   applications and jobs may crash.
+you.
 
-Example::
-
-   quota -s
-   Disk quotas for user vsc31234 (uid 123456):
-     Filesystem  space    quota   limit   grace   files   quota   limit   grace
-     nas2-ib1:/mnt/home
-                 648M   2919M   3072M            3685       0       0
-     nas2-ib1:/mnt/data
-               20691M  24320M  25600M            134k       0       0
-     nas1-ib1:/mnt/site_scratch
-                    0  24320M  25600M               1       0       0
-
-Each line represents a file system you have access to, ``$VSC_HOME``,
-``$VSC_DATA``, and, for this particular example, ``$VSC_SCRATCH_SITE``.
-When using the ``-s`` flag, ``quota`` will report disk space and limits
-in human-readable format, i.e., using MB or GB, rather than blocks.
-
-Some file systems have limits on the number of files that can be
-stored, and those are represented by the last four columns. The
-
-``files``
-   The number of files on the file system owned by you.
-``quota``
-   If the number of files exceeds this value, you may
-   get warnings (disabled on most systems).  The reported number
-   of files will be marked with an ``*`` if it exceeds the value
-   of ``quota``.
-``limit``
-   If the limit is reached, no new files can be written to disk, so
-   applications and jobs may crash.
-
-.. note::
-
-   Using these commands on another cluster than the one
-   in your home institution, will fail to return information
-   about the quota on your ``$VSC_HOME`` and ``$VSC_DATA``
-   directories and will show you  the quota for your ``$VSC_SCRATCH``
-   directory on that system.
+On most systems, ``myquota`` will show you for the ``$VSC_HOME``,
+``$VSC_DATA`` and ``$VSC_SCRATCH`` file systems either which
+percentage of the available disk space you are using, or the
+absolute amount. Users from Ghent university should check their disk usage
+using the `web application <https://account.vscentrum.be/>`_.
 
+If quota have been set on the number of files you can create on a file
```

Comment:
Technically, that would be inodes on GPFS, no? Which is not exactly the same.

Reply:
I think explaining the difference between inodes and actual file counts would only muddle matters.  The point is that people have a quick way to see they're using too much whatever.

I'd suggest to use "files" as proxy.